lib/AppInfo/Application.php

```diff
@@ -168,6 +169,24 @@ public function checkAndEnableCODEServer() {
 			$absoluteUrl = $urlGenerator->getAbsoluteURL($relativeUrl);
 			$wopi_url = $absoluteUrl . '?req=';
 
+            $CODEStatusService = $this->getContainer()->query(CODEStatusService::class);
+            $statusResponse = $CODEStatusService->checkCODEProxyStatus($wopi_url);
```

Comment:
Hm, so in case an error happens, it would retry on every request and perform a request to the proxy each time. Not sure how to avoid that :/